### PR TITLE
Update strings.po Spanish Language

### PR DESCRIPTION
**Pull request by gtkingbuild**

### CODE REVIEW

**Comment by icarok99** [comment]
Strings.po It is out of date and is missing some texts 

https://github.com/icarok99/skin.arctic.fuse/blob/omega/language/resource.language.es_es/strings.po

**Comment by icarok99** [comment]
> Strings.po It is out of date and is missing some texts
> 
> https://github.com/icarok99/skin.arctic.fuse/blob/omega/language/resource.language.es_es/strings.po

Create a 2 branch with your translation and then replace with that translation and you will check which texts are missing

File: language/resource.language.es_es/strings.po [hunk 1]
```diff
@@ -18,2197 +18,2340 @@ msgstr ""
...
 
 #: /1080i/Custom_1115_Dialog_Shortcuts.xml
 msgctxt "#31002"
 msgid "Edit $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] categories"
-msgstr ""
+<<<<<<< Updated upstream
+msgstr "Edit $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] categorias"
+=======
+msgstr "Editar $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] categorias"
+>>>>>>> Stashed changes
```
Comment (jurialmunkey) [inline language/resource.language.es_es/strings.po:35]: These need to be fixed

**Review by jurialmunkey — CHANGES_REQUESTED** [review]
Hi thanks for submitting this translation. There's quite a lot of diff type conflicts in the file that need to be resolves (all the "stash changes" lines etc.). IF those can be fixed up then I can merge.

**Comment by icarok99** [comment]
> Hi thanks for submitting this translation. There's quite a lot of diff type conflicts in the file that need to be resolves (all the "stash changes" lines etc.). IF those can be fixed up then I can merge.

Open another pull request with the changes.

**Comment by jurialmunkey** [comment]
Add in #551

**Pull request closed without merging**